### ISSUE TITLE
Squash Alembic revisions

### ISSUE THREAD

**Issue by shangyian** [DataJunction/dj#301]
At this point the data models have all changed enough that for maintenance/cleanliness, we should consider squashing the current alembic revisions into a single revision, as suggested here: https://github.com/DataJunction/dj/pull/289#pullrequestreview-1284549578. 

However, if anyone has an active version of DJ deployed somewhere, where newer revisions need to be relayed on top of an existing database, we can close this issue.

**Comment by betodealmeida** [comment]
Ah, good point... do you think we should wait until we have a 0.1 release? Or do it again by then?

**Comment by shangyian** [comment]
Good point! Let's wait until an 0.1 release. I got ahead of myself and created a PR, but then shortly afterwards started a branch that creates another revision... 😄
